### ISSUE TITLE
About protecting starting a session with the same name

### ISSUE THREAD

**Issue by borsatti2** [allburov/whatsapp-http-api#315]
You could create a system that if you try to create a session with the same name, it returns a failure, if the session exists and they are not active, then close it and open a new one, if there is one active (connected) then, do not perform any action

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Fixed in `2024.5.3` (WAHA Plus)

>if there is one active (connected) then, do not perform any action

If you perform two `POST /api/sessions` request with the same name, like
```json
{
  "name": "sessionname",
}
```
It return response with status code 422 and payload:

```json
{
  "statusCode": 422,
  "message": "Session 'default3' is already started.",
  "error": "Unprocessable Entity"
}
```

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
If the session exists and it's STOPPED - we restart it

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
